create_playlist.py

```diff
@@ -1,31 +1,33 @@
 # step 1 sign into youtube
 # step 2 create a new playlist album
-# step 3 grab like song on youtube 
+# step 3 grab like song on youtube
 # step 4 search for song
 # step 5 add song into album
 
-# PYTHON IS AN INTERPRETTED LANGUAGE (COMPILE BUTTON) AND OBJECT ORIENTED. 
+# PYTHON IS AN INTERPRETTED LANGUAGE (COMPILE BUTTON) AND OBJECT ORIENTED.
 
-import json # format for storing and transporting data.  A key and value data format to transfer data
+import json  # format for storing and transporting data.  A key and value data format to transfer data
 import requests  # an HTTP library  (communication from user to code)
 import os
 
 
-from secret import spotify_user_id,spotify_token  #takes the variable we want from secret file
+# takes the variable we want from secret file
+from secret import spotify_user_id, spotify_token
 import google_auth_oauthlib.flow
 import googleapiclient.discovery
 import googleapiclient.errors
 import youtube_dl
- 
+
+
 class CreatePlaylist:
-    
+
     def _init_(self):
-        self.user_id= spotify_user_id # takes the username in our secret file 
+        self.user_id = spotify_user_id  # takes the username in our secret file
         self.spotify_token = spotify_token
         self.get_youtube_client = self.get_youtube_client()
```

Comment:
If you assign something to `self.get_youtube_client`, you will overwrite the `get_youtube_client` method.
Based on [line 55](https://github.com/jcraay/SpotifyPlaylist/pull/1/files#diff-bbd112e5b30074d9a647b51925de6f81R55) I think you may have meant to call this `self.youtube_client`.